doc/setup_assistant/setup_assistant_tutorial.rst

```diff
@@ -275,6 +275,26 @@ files that you will need to start using MoveIt!
 * Congratulations!! - You are now done generating the configuration
   files you need for MoveIt!
 
+
+Gazebo Simulation
+------------------------------------
+
+You can use the generated robot urdf file in the simulation folder to
+spawn the robot in Gazebo.
+
+* Use rosrun to start gazebo empty world: ::
```

Comment:
Make ``rosrun`` have single quotes (`) around it so it looks like a code command

Reply:
ping: these requested changes

Reply:
Done